flow/record/adapter/splunk.py

```diff
@@ -27,7 +50,18 @@
 )
 
 
-def splunkify(record, tag=None):
+class SPLUNK_PROTOCOLS(Enum):
```

Comment:
```suggestion
class Protocol(Enum):
```
Unless there's a specific reason to prefix it with `Splunk`, I believe it being in this file should be context enough. Also enum classes are usually singular and Camelcased, with the values UPPER_CASE.